### PR TITLE
docs(v2): add unofficial SFMC Code Doc to Showcase

### PR DESCRIPTION
**Pull request by MateuszDabrowski**
## Motivation

Showcase of another page created using Docusaurus v2 - this time multi doc page focused on Salesforce Marketing Cloud Programmatic Languages.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

N/A

## Related PRs

N/A

## Screenshot of the page for Showcase

md

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -258,6 +258,15 @@ const users = [
     fbOpenSource: false,
     pinned: true,
   },
+  {
+    title: 'MateuszDabrowski',
+    description: 'Salesforce Marketing Cloud Programmatic Documentation',
+    preview: require('./showcase/md.png'),
```
Comment (yangshun) [inline website/src/data/users.js:264]: Don't see this file being added

**Review by yangshun — CHANGES_REQUESTED** [review]
Please fix the missing files.

File: website/src/data/users.js (hunk 1 above)
Comment (MateuszDabrowski) [reply website/src/data/users.js:264]: I see information about lack of privileges to add any files into the folder containing screenshots, that’s why I added it to this ticket.

Comment (yangshun) [reply website/src/data/users.js:264]: You have to fork the repository, and add the image to your fork.

File: website/src/data/users.js [hunk 2]
```diff
@@ -258,6 +258,15 @@ const users = [
     fbOpenSource: false,
     pinned: true,
   },
+  {
+    title: 'MateuszDabrowski',
+    description: 'Salesforce Marketing Cloud Programmatic Documentation',
```
Comment (yangshun) [inline website/src/data/users.js:263]: Is this description accurate? It's perfectly okay to describe this site as your personal branding site and we rather give accurate descriptions.

Comment (MateuszDabrowski) [reply website/src/data/users.js:263]: I was thinking about the proper description and wanted it to showcase the main value behind the page for the potential viewer. I might propose an updated version with `'Personal branding site with unofficial Salesforce Marketing Cloud Programmatic Documentation'`. I believe that the main content on the site is focused on JS/SSJS and soon SFMC SQL docs, not personal items. Let me know if you would prefer the description proposed above.

**Comment by slorber** [comment]
closing for now, but feel free to reopen a proper PR including the image.

I would suggest that if the site has your name as a logo would should put your name clearly on the description.
Also the pinned option should rather be false unless we agree to highlight your site

**Pull request closed without merging**